test/e2e/logs/agent/severity_parser_test.go

```diff
@@ -21,10 +21,25 @@ func TestSeverityParser(t *testing.T) {
 	suite.RegisterTestCase(t, suite.LabelLogAgent)
 
 	var (
-		uniquePrefix = unique.Prefix()
-		pipelineName = uniquePrefix()
-		backendNs    = uniquePrefix("backend")
-		genNs        = uniquePrefix("gen")
+		uniquePrefix      = unique.Prefix()
+		pipelineName      = uniquePrefix()
+		backendNs         = uniquePrefix("backend")
+		genNs             = uniquePrefix("gen")
+		levelINFOScenario = map[string]string{
+			"scenario": "level-info",
+			"level":    "INFO",
+		}
+		levelWarningScenario = map[string]string{
+			"scenario": "level-warning",
+			"level":    "warning",
+		}
+		logLevelScenario = map[string]string{
+			"scenario":  "log.level",
+			"log.level": "WARN",
+		}
```

Comment:
is there a reason for the capitalization difference?

Reply:
I capitalized now the level in the `levelWarningScenario`, so all levels now are capitalized in the test